databases/week1/week1_HW.sql

```diff
@@ -0,0 +1,68 @@
+use hyf_class2023;
+
+-- 1 Find out how many tasks are in the task table
+
+SELECT count(id)
+FROM task;
+
+-- 2 Find out how many tasks in the task table do not have a valid due date
+
+SELECT count(id)
+FROM task
+where due_date is null;
+
+-- 3 Find all the tasks that are marked as done
+
+select title, status_id
+from task
+where status_id='3';
+
+-- 4 Find all the tasks that are not marked as done
+
+select title, status_id
+from task
+where status_id = '1' or status_id = '2';
+
+-- 5 Get all the tasks, sorted with the most recently created first
+
+select *
+from task
+order by created desc;
+
+-- 6 Get the single most recently created task
+
+select *
+from task
+order by created desc
+ limit 1;
+
+-- 7 Get the title and due date of all tasks where the title or description contains database
+
+select title
+, due_date
+from task
+ where
+(title like '%database%')
+ or
+(title like '%database%');
```

Comment:
line # 47 can be modified as 
`(description like %database%);`

Reply:
ahhh yes i missed that part 🤦🏽‍♂️